Makefile

```diff
@@ -286,7 +285,7 @@ vendor-in-container:
 ###
 
 # Make sure to warn in case we're building without the systemd buildtag.
-bin/podman: .gopathok $(SOURCES) go.mod go.sum
+bin/podman: .gopathok test/version/version $(SOURCES) go.mod go.sum
```

Comment:
Why do I have to build this for podman? I think this is only required for `podman-release.tar.gz` and `podman-remote-release-%.zip`

Reply:
100% concur, `go build` is slow as molasses and I'd prefer not to add another one to the development critical path of `make`.

Reply:
> I think this is only required for podman-release.tar.gz

That tarball includes both podman and podman-remote :cry: 

> go build is slow as molasses

Oh?  At least for my local testing, I didn't even notice the additional `version` build time.  Anyway, there _was_ a reason I added it here...but now I can't remember :disappointed:   I'll try taking it off and see what breaks (if anything).

Reply:
Oh! Now I remember.  The `podman-msi` target makes use of `$(RELEASE_NUMBER)` in the target name.  That was causing issues if the binary didn't exist when the makefile runs.  I think I know how I can work around that though...